docs/apache-airflow/howto/operator/python.rst

```diff
@@ -28,6 +28,16 @@ Use the ``@task`` decorator to execute Python callables.
     The ``@task`` decorator is recommended over the classic :class:`~airflow.operators.python.PythonOperator`
     to execute Python callables.
 
+.. note::
```

Comment:
Why here? It is importand and valid potint to mention but it works the same way for every other operator, so I guess it should be explained in https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/executor/local.html#local-executor

Reply:
@potiuk I missed that. I have made the changes in local.rst now. Let me know if that looks good

Reply:
@potiuk Updated now